doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/layouts/HStackExamples.swift

```diff
@@ -0,0 +1,227 @@
+import SwiftUI
+
+struct HStackLeftAlignedPage : View {
+    var body: some View {
+        NavigationStack {
+            HStack() {
+                Text("First")
+                Text("Second")
+                Text("Third")
+                Spacer()
+            }
+        }.navigationTitle("Left Aligned")
+    }
+}
+
+struct HStackCenterAlignedPage : View {
+    var body: some View {
+        NavigationStack {
+            HStack() {
+                Text("First")
+                Text("Second")
+                Text("Third")
+            }
+        }.navigationTitle("Center Aligned")
+    }
+}
+
+
+struct HStackRightAlignedPage : View {
+    var body: some View {
+        NavigationStack {
+            HStack() {
+                Spacer()
+                Text("First")
+                Text("Second")
+                Text("Third")
+            }.navigationTitle("Right Aligned")
+        }
+    }
+}
+
+struct HStackContentTopAlignedPage : View {
+    var body: some View {
+        NavigationStack {
+            HStack(alignment:.top) {
+                Text("First").font(.largeTitle)
+                Text("Second").font(.title)
+                Text("Third").font(.title2)
+            }.border(Color.blue, width: 1)
```

Comment:
What's the purpose for the `border()` in these examples? Is there a relationship between the definition of `HStack` and `border()`?

Reply:
`border()` was added just to add visual clarity how the `HStack` content alignment looks like. Kind of used as reference. It is used in Apple docs as well. Mostly border or frames are used. Removed for now.
Ref - 
https://developer.apple.com/documentation/swiftui/verticalalignment
https://developer.apple.com/documentation/swiftui/horizontalalignment